dts/msm8916/msm8916-samsung-r07.dts

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+    // This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <206 0>;
+	qcom,board-id = <0xCE08FF01 7>;
```

Comment:
It looks like `msm8916-sec-fortuna-r07.dts` contains two board-ids:

```
qcom,board-id = <0xCE08FF01 7>, <0xF808FF01 7>;
```

The `0xF808FF01` one is the appropriate one for 3g-only devices, so please rename this file to `msm8216-samsung-r07.dts` and use the `<0xF808FF01 7>` board-id.